galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/IDockerContainerConfig.java

```diff
@@ -37,4 +37,18 @@
      */
     public IDockerVolume getVolumeByTag(String volumeTag);
     
+
+    /**
+     * Allows a set of ports to be exposed before container startup.
+     * 
+     * @param ports
+     */
+    public void setExposedPorts(List<String> ports);
```

Comment:
Exposed ports need 1 or 2 port numbers,    the one for the host system and the one in the container.   ie 8080/tcp->80/tcp
You will need to implement it properly I am afraid.